rudder-agent/SOURCES/patches/fusioninventory/11726-Extend-Rudder-specific-inventory-with-clie.patch

```diff
@@ -0,0 +1,106 @@
+From e13d5499713d61385843c8c272136c019dabad3f Mon Sep 17 00:00:00 2001
+From: Nicolas Charles <nicolas.charles@normation.com>
+Date: Tue, 14 Nov 2017 14:51:26 +0100
+Subject: [PATCH 1/1] Fixes #403: Extend Rudder specific inventory with client
+ side data
+
+---
+ lib/FusionInventory/Agent/Inventory.pm             |  2 +-
+ .../Agent/Task/Inventory/Generic/Rudder.pm         | 48 ++++++++++++++++++++++
+ 2 files changed, 49 insertions(+), 1 deletion(-)
+
+diff --git a/lib/FusionInventory/Agent/Inventory.pm b/lib/FusionInventory/Agent/Inventory.pm
+index 7467a30..08a7fa6 100644
+--- a/lib/FusionInventory/Agent/Inventory.pm
++++ b/lib/FusionInventory/Agent/Inventory.pm
+@@ -77,7 +77,7 @@ my %fields = (
+                              CMD/ ],
+     REGISTRY         => [ qw/NAME REGVALUE HIVE/ ],
+     REMOTE_MGMT      => [ qw/ID TYPE/ ],
+-    RUDDER           => [ qw/AGENT UUID HOSTNAME SERVER_ROLES AGENT_CAPABILITIES/ ],
++    RUDDER           => [ qw/AGENT UUID HOSTNAME SERVER_ROLES AGENT_CAPABILITIES CUSTOM_PROPERTIES/ ],
+     SLOTS            => [ qw/DESCRIPTION DESIGNATION NAME STATUS/ ],
+     SOFTWARES        => [ qw/COMMENTS FILESIZE FOLDER FROM HELPLINK INSTALLDATE
+                             NAME NO_REMOVE RELEASE_TYPE PUBLISHER
+diff --git a/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm b/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm
+index b2fd04d..a36e450 100644
+--- a/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm
++++ b/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm
+@@ -7,6 +7,8 @@ use English qw(-no_match_vars);
+ 
+ use FusionInventory::Agent::Tools;
+ 
++use JSON::PP;
```

Comment:
Do we have to install this as a new dependency ?

Reply:
no, this is already included and used by agent